src/MongoDB.Driver/Encryption/MongocryptdFactory.cs

```diff
@@ -165,6 +165,33 @@ private bool ShouldMongocryptdBeSpawned(out string path, out string args)
             }
 
             return false;
+
+            string GetMongocryptdExtension()
+            {
+                var currentOperatingSystem = OperatingSystemHelper.CurrentOperatingSystem;
+                switch (currentOperatingSystem)
+                {
+                    case OperatingSystemPlatform.Windows:
+                        // "nul" is the windows specific value.
+                        return ".exe";
+                    case OperatingSystemPlatform.Linux:
+                    case OperatingSystemPlatform.MacOS:
+                    default: return string.Empty;
```

Comment:
Formatting is a bit wonky. `return string.Empty;` should be on its own line. And stylistically, we should be using `""` rather than `string.Empty`.

Reply:
done